src/domains/citizen-auth-common/99_locals.tf

```
@@ -33,4 +33,16 @@ locals {
 
   vnet_common_name_itn                = "${local.common_project_itn}-common-vnet-01"
   vnet_common_resource_group_name_itn = "${local.common_project_itn}-common-rg-01"
+
+  location           = "westeurope"
+  secondary_location = "italynorth"
+
+  itn_environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location_short
```

location contains the full name of the azure region. In this case, with this itn_environment local, we want to address resources created in italynorth. We can use the existing itn_location local

```suggestion
    location        = local.itn_location
```